src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.GateThread.cs

```diff
@@ -14,8 +14,8 @@ private static class GateThread
             private const int GateThreadDelayMs = 500;
             private const int DequeueDelayThresholdMs = GateThreadDelayMs * 2;
             
-            private static bool s_disableStarvationDetection = true; // TODO: Config
-            private static bool s_debuggerBreakOnWorkStarvation = false; // TODO: Config
+            private static readonly bool s_disableStarvationDetection = (bool?)AppContext.GetData("ThreadPool_DisableStarvationDetection") ?? true;
+            private static readonly bool s_debuggerBreakOnWorkStarvation = (bool?)AppContext.GetData("ThreadPool_DebugBreakOnWorkerStarvation") ?? false;
```

Comment:
Since we're supporting string values elsewhere, should support them here too. Would be good to use a common helper that takes a default value